sdk/mongocluster/arm-mongocluster/test/public/node/privateEndpointConnections.spec.ts

```diff
@@ -65,18 +65,15 @@ describe("MongoCluster test", () => {
       {
         location,
         properties: {
-          administratorLogin: "mongoAdmin",
-          administratorLoginPassword: "SecureString;",
-          nodeGroupSpecs: [
-            {
-              diskSizeGB: 128,
-              enableHa: true,
-              kind: "Shard",
-              nodeCount: 1,
-              sku: "M30",
-            },
-          ],
+          administrator: {
+            userName: "mongoAdmin",
+            password: "SecureString;",
+          },
           serverVersion: "5.0",
+          storage: { sizeGb: 128 },
```

Comment:
why we have this change?

could you remind me why we have these cases in node folder?

Reply:
their ga version has the property change
https://github.com/kazrael2119/azure-rest-api-specs/blob/c3474b5f97987003f73a08765b3450f2829eb6ea/specification/mongocluster/DocumentDB.MongoCluster.Management/MongoCluster.tsp#L212-L218